packages/devtools/kit/src/client.ts

```diff
@@ -32,12 +32,21 @@ export class AssetDefinition {
   logo: string = logo;
 }
 
+export class AnnouncementDefinition {
+  src: string =
+    'https://raw.githubusercontent.com/web-infra-dev/modern.js/main/ANNOUNCEMENT.md';
```

Comment:
Why is ANNOUNCEMENT a markdown file rather than JSON

Reply:
As a top-level file of the repository, I would like to make it more readable although it may lead to little parse cost.